README.md

```diff
@@ -0,0 +1,39 @@
+# OpenCompose
+
+[![Build Status](https://travis-ci.org/redhat-developer/opencompose.svg?branch=master)](https://travis-ci.org/redhat-developer/opencompose)
+
+The goal of OpenCompose is to make it easier to on-board developers to Kubernetes. It is a curated higher level abstraction for specific Kubernetes resources. Very simple idea, isn't it? Developer shouldn't have to learn various Kubernetes concepts just to test and deploy their applications on Kubernetes. The developer is generally concerned with the application that is being developed.
+
+We are at a very nascent stage of this project and we have certain [ideas](https://github.com/redhat-developer/opencompose/blob/master/examples/basic.yaml) but we are definitely looking for suggestions and contributions from the Kubernetes community. Please send PRs to improve this project,
+file issues and RFEs against this repository.
+
+### Installation
+We don't have release yet, but you can easily build OpenCompose tool from source.
+All you need is [Go installed](https://golang.org/dl/) on your system. Than simply run:
+```
+go get -u https://github.com/redhat-developer/opencompose
+```
+
+### Example
+1) Create or download [hello-nginx.yml](https://github.com/redhat-developer/opencompose/blob/master/examples/hello-nginx.yaml) file.
+
+```yaml
+version: 0.1-dev
+services:
+- name: helloworld
+  containers:
+  - image: tomaskral/nonroot-nginx
+    ports:
+    - port: 8080
+```
+
+2) Convert OpenCompose file to Kubernetes objects
+```
+opencompose convert -f examples/hello-nginx.yml
+```
+
+This will create two files in current directory - *deployment-helloworld.yaml* and *service-helloworld.yaml*.
+First one is Kubernetes Deployment object and second is Kubernetes Service.
+If you want deploy this to Kubernetes cluster just run `kubectl create -f service-helloworld.yaml -f deployment-helloworld.yaml`
+
```

Comment:
I would look more consistent to have the kubectl line look line those code examples above with '```'